Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAMSIConfigurationState.ps1

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\..\Helpers\Invoke-CatchActions.ps1
+Function Get-ExchangeAMSIConfigurationState {
+
+    begin {
+        Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+    } process {
+        try {
+            Write-Verbose "Trying to query AMSI configuration state"
+            $amsiConfiguration = Get-SettingOverride | Where-Object { ($_.ComponentName -eq "Cafe") -and ($_.SectionName -eq "HttpRequestFiltering") } -ErrorAction Stop
+
+            if (($null -ne $amsiConfiguration) -and
+                ($amsiConfiguration.Count -eq 1)) {
+                Write-Verbose "Setting override detected for AMSI configuration"
+                $amsiConfigurationQuerySuccessful = $true
```

Comment:
Move `$amsiConfigurationQuerySuccessful = $true` to right after `Get-SettingOverride`, no need to have it set to `true` in all the if statements

Reply:
Done. Will be addressed with the next commit.